packages/@aws-cdk/aws-scheduler-alpha/lib/group.ts

```diff
@@ -0,0 +1,363 @@
+      grantee: identity,
+      actions: ['scheduler:GetSchedule', 'scheduler:ListSchedules'],
+      resourceArns: [this.arnForScheduleInGroup('*')],
+      scope: this,
+    });
+  }
+
+  /**
+   * Grant create and update schedule permissions for schedules in this group to the given principal
+   */
+  grantWriteSchedules(identity: iam.IGrantable): iam.Grant {
+    return iam.Grant.addToPrincipal({
+      grantee: identity,
+      actions: ['scheduler:CreateSchedule', 'scheduler:UpdateSchedule'],
+      resourceArns: [this.arnForScheduleInGroup('*')],
+      scope: this,
+    });
+  }
+
+  /**
+   * Grant delete schedule permission for schedules in this group to the given principal
+   */
+  grantDeleteSchedules(identity: iam.IGrantable): iam.Grant {
+    return iam.Grant.addToPrincipal({
+      grantee: identity,
+      actions: ['scheduler:DeleteSchedule'],
+      resourceArns: [this.arnForScheduleInGroup('*')],
+      scope: this,
+    });
+  }
+}
+
+export class Group extends GroupBase {
+  /**
+   * Import an external group by ARN.
+   *
+   * @param scope construct scope
+   * @param id construct id
+   * @param groupArn the ARN of the group to import (e.g. `arn:aws:scheduler:region:account-id:schedule-group/group-name`)
+   */
+  public static fromGroupArn(scope: Construct, id: string, groupArn: string): IGroup {
+    const arnComponents = Stack.of(scope).splitArn(groupArn, ArnFormat.SLASH_RESOURCE_NAME);
+    const groupName = arnComponents.resourceName!;
+    class Import extends GroupBase {
+      public groupName = groupName;
+      public groupArn = groupArn;
+    }
+    return new Import(scope, id);
+  }
+
+  /**
+   * Import a default schedule group.
+   *
+   * @param scope construct scope
+   * @param id construct id
+   */
+  public static fromDefaultGroup(scope: Construct, id: string): IGroup {
+    return Group.fromGroupName(scope, id, 'default');
+  }
+
+  /**
+   * Import an existing group with a given name.
+   *
+   * @param scope construct scope
+   * @param id construct id
+   * @param groupName the name of the existing group to import
+   */
+  static fromGroupName(scope: Construct, id: string, groupName: string): IGroup {
+    const groupArn = Stack.of(scope).formatArn({
+      service: 'scheduler',
+      resource: 'schedule-group',
+      resourceName: groupName,
+    });
+    return Group.fromGroupArn(scope, id, groupArn);
+  }
+
+  public readonly groupName: string;
+  public readonly groupArn: string;
+
+  constructor(scope: Construct, id: string, props: GroupProps) {
+    super(scope, id, {
+      physicalName: props.groupName ?? PhysicalName.GENERATE_IF_NEEDED,
+    });
+
+    const group = new CfnScheduleGroup(this, 'Resource', {
+      name: this.physicalName,
+    });
+
+    this.groupArn = this.getResourceArnAttribute(group.attrArn, {
+      service: 'scheduler',
+      resource: 'schedule-group',
+      resourceName: this.physicalName,
+    });
+    this.groupName = this.physicalName;
+  }
```

Comment:
I think a call to applyRemovalPolicy is missing here? This means the removalPolicy from the props will not be emitted to CloudFormation.

Reply:
Thanks! I have actually missed that.

> @default RemovalPolicy.RETAIN

Are we sure that we want the default to be `RETAIN`?